.ci/test_r_package_windows.ps1

```diff
@@ -28,6 +28,24 @@ function Run-R-Code-Redirect-Stderr {
   Rscript --vanilla -e $decorated_code
 }
 
+# Remove all items matching some pattern from PATH environment variable
+function Remove-From-Path {
+  param(
+    [string]$item_to_remove
+  )
+  $env:PATH = ($env:PATH.Split(';') | Where-Object { $_ -notmatch "$item_to_remove" }) -join ';'
```

Comment:
I think since both of your suggestions were on the same line, one overwrote the other when I accepted them in the browser.

```suggestion
    [string]$pattern_to_remove
  )
  $env:PATH = ($env:PATH.Split(';') | Where-Object { $_ -notmatch "$pattern_to_remove" }) -join ';'
```

Reply:
Oh, sorry about that! Next time I'll merge all suggestions for the same lines into a single one.